coalib/bearlib/abstractions/Linter.py

```diff
@@ -444,9 +434,34 @@ def process_output_regex(
                                 "result_message")
                     if key in options}
 
-                process_output = partialmethod(
+                _processing_function = partialmethod(
                     process_output_regex, **_process_output_args)
 
+            def process_output(self, output, filename, file):
+                """
+                Processes the output of the executable and yields results
+                accordingly.
+
+                :param output:
+                    The output of the executable. This can be either a string
+                    or a tuple depending on the usage of ``use_stdout`` and
+                    ``use_stderr`` parameters of ``@linter``. If only one of
+                    these arguments is ``True``, a string is placed (containing
+                    the selected output stream). If both are ``True``, a tuple
+                    is placed with ``(stdout, stderr)``.
+                :param filename:
+                    The name of the file currently processed.
+                :param file:
+                    The contents of the file (line-splitted).
+                """
+
```

Comment:
I'd personally remove this newline but that's a taste thing, keep if if you want